### PR TITLE
Upgraded NetApp SDK to the latest 1.13.0

### PR DESCRIPTION
**Pull request by AdelGhabboun**
1. Upgraded ANF SDK to the latest 1.13.0
2. Upgraded target framework to .NET 5.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
 ## Azure NetApp Files SDK Sample - Snapshot Policy for .NETCore
 
+[1.0.1 (2021-09-02)](#1.0.1 (2021-09-02))
 [1.0.0 (2020-11-02)](#1.0.0 (2020-11-02))
 
 # 1.0.0 (2020-11-02)
```
Comment (paulomarquesc) [inline CHANGELOG.md]: Version and date needs to be updated #Resolved

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
 
+*Features*
+* Upgraded NetApp Files SDK version to 1.13.0
```
Comment (paulomarquesc) [inline CHANGELOG.md:9]: >* U [](http://example.com/codeflow?start=0&length=3)

Missing .net 5 update #Resolved

File: src/anf-dotnetcore-snapshot-policy-sdk-sample/Program.cs [hunk 3]
```diff
@@ -123,7 +122,8 @@ private static async Task CreateANFAsync()
                 location,
                 anfAccountName,
                 snapshotPolicyName,
-                newSnapshotPolicy.HourlySchedule);            
+                newSnapshotPolicy.HourlySchedule);
+            //await WaitForAnfResource<SnapshotPolicy>(anfClient, currentSnapshotPolicy.Id);
```
Comment (paulomarquesc) [inline src/anf-dotnetcore-snapshot-policy-sdk-sample/Program.cs]: >//await WaitForAnfResource<SnapshotPolicy>(anfClient, currentSnapshotPolicy.Id); [](http://example.com/codeflow?start=12&length=80)

Is there a reason why this is commented out? #Resolved

**Review by paulomarquesc — CHANGES_REQUESTED** [review]
:clock1:

File: CHANGELOG.md (hunk 1 above)
Comment (AdelGhabboun) [reply CHANGELOG.md]: It is update d, isn't it ?

Comment (AdelGhabboun) [reply CHANGELOG.md]: oh found it, no worries

File: src/anf-dotnetcore-snapshot-policy-sdk-sample/Program.cs (hunk 3 above)
Comment (AdelGhabboun) [reply src/anf-dotnetcore-snapshot-policy-sdk-sample/Program.cs]: was part of testing and forgot to uncomment it

**Review by paulomarquesc — APPROVED** [review]
:shipit:

**Pull request merged**